File: Data Structures & Algorithms/anagram-groups/submission-0.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        unordered_map<string, vector<string>> sortMap;
        for(int i=0; i<strs.size(); i++){
            string sorted = strs[i];
            sort(sorted.begin(), sorted.end());
            sortMap[sorted].push_back(strs[i]);
        }
        vector<vector<string>> result;
        for(const auto& p : sortMap){
            result.push_back(p.second);
        }
        return result;
    }
};

File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        result = []
        s = []
        for i in range(len(temperatures)):
            result.append(0)
            for j in range(i, len(temperatures)):
                if(temperatures[j]>temperatures[i]):
                    result[i] = j-i
                    break
            
        return result
                
                
        
        